### ISSUE TITLE
Error on creating new file (intermittent)

### ISSUE THREAD

**Issue by AdrianoFerrari** [gingko/client#138]
Occasionally, this error appears on opening a new document:
```
Unhandled Promise Rejection
Error: Cannot find module '../screen'
    at Function.Module._resolveFilename (/tmp/.mount_gingkoIzXuqN/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Object.get [as screen] (/tmp/.mount_gingkoIzXuqN/resources/electron.asar/browser/api/exports/electron.js:11:16)
    at module.exports (/tmp/.mount_gingkoIzXuqN/resources/app.asar/electron.js:227842:25)
    at createDocumentWindow (/tmp/.mount_gingkoIzXuqN/resources/app.asar/electron.js:126659:25)
    at newUntitled (/tmp/.mount_gingkoIzXuqN/resources/app.asar/electron.js:126942:3)
    at <anonymous>
```

Can't reliably reproduce it. Might be related to making app single instance, and this electron bug:
https://github.com/electron/electron/issues/8862

**Comment by AdrianoFerrari** [comment]
Possibly resolved in 316962f , after upgrading to electron 3.0

**Comment by AdrianoFerrari** [comment]
I've finally been able to reproduce this reliably. It happens only **when the internet is disconnected**.
Looking into a fix.
